src/vrcosclib/Utility/OscUtility.Connection.cs

```diff
@@ -45,39 +46,34 @@ public static int ReceivePort
         }
     }
 
-    private static int _sendPort = 9000;
     public static int SendPort
     {
-        get => _sendPort;
+        get => _destination.Port;
         set
         {
-            if (value > 65535 || value < 0)
-            {
-                throw new ArgumentOutOfRangeException(nameof(value));
-            }
-            if (_sendPort == value)
-            {
-                return;
-            }
-
-            _sendPort = value;
-            if (_client != null)
-            {
-                _client.Dispose();
-                _client = null;
-            }
+            //Do we need this setter? Doesn't look so good...
+            Destination = new IPEndPoint(Destination.Address, value);
         }
     }
 
-    private static string _clientIPAddress = "127.0.0.1";
+    public static string VrcIPAddress
```

Comment:
I felt that `VrcIPAddress` was unnecessary if there was `Destination`. No need to change, but it may be removed before release.